Update trial conditions for service inactivity (#4132)
timescale/docs: about/pricing-and-account-management.md

We offer new users a free, 30-day trial period of our $PERFORMANCE plan with no credit card required.  
During your trial, you can contact sales@timescale.com to request information about, and access

> to, our $SCALE plan to determine how it fits your needs. During your trial, $SERVICE_SHORTs that receive no queries for 7 days are paused without any data loss, but you can resume them in $CONSOLE.

Reviewer: ```suggestion
to, our $SCALE plan to determine how it fits your needs. During your trial, $SERVICE_SHORTs that receive no queries for 7 days are paused without any data loss. You easily resume them in $CONSOLE.
```

After you have completed your 30-day trial period on the $PERFORMANCE plan, choose the 

> [$PRICING_PLAN][plan-features] that suits your business and engineering needs. Once you become a paying user, we can make

Reviewer: ```suggestion
[$PRICING_PLAN][plan-features] that suits your business and engineering needs. 

After you become a paying user, we can enable some features in the higher $PRICING_PLANs so you can test them before upgrading.
```

We offer new users a free, 30-day trial period of our $PERFORMANCE plan with no credit card required.  
During your trial, you can contact sales@timescale.com to request information about, and access

> to, our $SCALE plan to determine how it fits your needs. During your trial, $SERVICE_SHORTs that receive no queries for 7 days are paused without any data loss. You easily resume them in $CONSOLE.

Reviewer 2: think we could improve it like this:

> If a $SERVICE_SHORT doesn’t receive any queries for 7 days, it will be paused to conserve resources. Your data remains intact during the trial, and you can easily resume the service from $CONSOLE. After your trial ends, we may remove your data unless you’ve added a payment method.
